common/types/identity_state.go

```diff
@@ -0,0 +1,38 @@
+package types
+
+import (
+	"errors"
+	"fmt"
+)
+
+var ErrIdentityStateUnknown = errors.New("identity state is unknown")
+
+type IdentityState int
+
+const (
+	IdentityStateNotSet IdentityState = iota
+	IdentityStateWaitForATXSyncing
+	IdentityStateWaitForPoetRoundStart
+	IdentityStateWaitForPoetRoundEnd
+	IdentityStateFetchingProofs
+	IdentityStatePostProving
+)
```

Comment:
I feel like this should also be part of the activation package - only the `activation` package and `grpcserver` needs them and the later already imports the former.